decentraland/unity-renderer
# fix: user profile activation date label

## What does this PR change?

`Fixes #1651 `
"Active since {month} {year}" showed at PlayerProfileInfoCard was not getting refreshed accordingly to the user's activation date. This changes fixes this problem. Kernel was not providing this data from the `UserProfileBridge`, so the solution was fetching the user profile data directly from the catalyst, parsing the `timestamp` considered as the profile's activation date.

## How to test the changes?

1. Go to: https://play.decentraland.zone/?renderer-branch=fix/profile-active-since
2. Click on the users around you to show the player's info card
3. In case the user has an activation data set, you should see a label indicating the date it was activated. ie: _Active since December 2021_
4. In case the user has no data about activation date, the label should be hidden

## Our Code Review Standards

https://github.com/decentraland/unity-renderer/blob/master/docs/code-review-standards.md
